src/main/java/pl/edu/icm/pl/mxrdr/extension/dataset/tab/DatasetAnalysisTab.java

```diff
@@ -0,0 +1,175 @@
+
+import com.google.common.collect.Lists;
+
+import edu.harvard.iq.dataverse.DatasetFieldServiceBean;
+import edu.harvard.iq.dataverse.DataverseRequestServiceBean;
+import edu.harvard.iq.dataverse.PermissionsWrapper;
+import edu.harvard.iq.dataverse.persistence.dataset.Dataset;
+import edu.harvard.iq.dataverse.persistence.dataset.DatasetField;
+import edu.harvard.iq.dataverse.persistence.dataset.DatasetFieldType;
+import edu.harvard.iq.dataverse.persistence.dataset.DatasetVersion;
+import edu.harvard.iq.dataverse.persistence.workflow.WorkflowArtifact;
+import edu.harvard.iq.dataverse.persistence.workflow.WorkflowExecution;
+import edu.harvard.iq.dataverse.settings.SettingsServiceBean;
+import edu.harvard.iq.dataverse.workflow.artifacts.WorkflowArtifactServiceBean;
+import edu.harvard.iq.dataverse.workflow.execution.WorkflowContext.TriggerType;
+import edu.harvard.iq.dataverse.workflow.execution.WorkflowExecutionServiceBean;
+
+@ViewScoped
+@Named("datasetAnalysisTab")
+public class DatasetAnalysisTab implements Serializable {
+    
+    private static final Logger logger = LoggerFactory.getLogger(DatasetAnalysisTab.class);
+
+    private DatasetFieldServiceBean datasetFields;
+    
+    private SettingsServiceBean settingsService;
+    
+    private PermissionsWrapper permissionsWrapper;
+
+    private DataverseRequestServiceBean dvRequestService;
+    
+    private WorkflowExecutionServiceBean workflowServiceBean;
+    
+    private WorkflowArtifactServiceBean workflowArfifactService;
+
+    @Inject
+    public DatasetAnalysisTab(SettingsServiceBean settingsService, PermissionsWrapper permissionsWrapper,
+            DataverseRequestServiceBean dvRequestService, WorkflowExecutionServiceBean workflowServiceBean,
+            WorkflowArtifactServiceBean workflowArtifactService, DatasetFieldServiceBean datasetFields) {
+        this.settingsService = settingsService;
+        this.permissionsWrapper = permissionsWrapper;
+        this.dvRequestService = dvRequestService;
+        this.workflowServiceBean = workflowServiceBean;
+        this.workflowArfifactService = workflowArtifactService;
+        this.datasetFields = datasetFields;
+
+    }
+    
+    // -------------------- GETTERS --------------------
+
+    
+    public List<AnalysisResultDTO> getAnalysisFields(DatasetVersion datasetVersion) {
+        List<AnalysisResultDTO> resultList = new ArrayList<AnalysisResultDTO>();
+        for (XdsAnalysisField field:XdsAnalysisField.values()) {
+            AnalysisResultDTO res = new AnalysisResultDTO();
+            DatasetFieldType type = datasetFields.findByName(field.getName());
+            res.setFieldLabel(type != null ? type.getLocaleTitle() : field.getName());
+
+            Optional<DatasetField> optional = datasetVersion.getDatasetFieldByTypeName(field.getName());
+            if (optional.isPresent()) {
+                DatasetField datasetField = optional.get();
+                res.setPrimaryValue(datasetField.getValue());
```

Comment:
maybe getDisplayValue would be better

Reply:
`getDisplayValue` - For top level fields (without parent) it would be probably ok. But when we operate on children fields then for some fields we would got something bad. If child field would have some displayFormat then it would render badly. Some fields in mxrdr is defined as `#NAME #VALUE`. For this fields we would render unnecessarily name of the field in table cell.

Reply:
Couldn't find any suitable getter, so I wrote my own one